src/ducks/balance/components/BalanceHeader.jsx

```diff
@@ -39,7 +40,9 @@ const BalanceHeader = ({
         onClickBalance={onClickBalance}
       />
       {accounts && (
-        <History accounts={accounts} transactions={transactionsCollection} />
+        <Delayed delay={500}>
+          <History accounts={accounts} transactions={transactionsCollection} />
+        </Delayed>
```

Comment:
I more 100% sure it's related, but when testing, I get the bug that was previously resolved by https://github.com/cozy/cozy-banks/pull/1510. Maybe the fix is not strong enough :(

Reply:
I fixed the line bug in https://github.com/cozy/cozy-banks/pull/1556/commits/a318c8383f3649b48abf250aaa150acba8407d6b (but it is hacky...). The tooltip placement is fixed in #1564 (less hacky).